mod/discussions/views/default/resources/discussion/owner.php

```diff
@@ -21,11 +26,19 @@
 	'subtype' => 'discussion',
 	'limit' => max(20, elgg_get_config('default_limit')),
 	'order_by' => 'e.last_action desc',
-	'container_guid' => $guid,
 	'full_view' => false,
 	'no_results' => elgg_echo('discussion:none'),
 	'preload_owners' => true,
 );
+
+if ($target instanceof ElggUser) {
+	// Display all discussions started by the user regardless of
+	// the entity that is working as a container. See #4878.
+	$options['owner_guid'] = $guid;
```

Comment:
@hypeJunction I renamed $owner to $target and made this change, so now users will be able to see all discussions they have started.